lang/en.json

```diff
@@ -396,5 +400,6 @@
   "YZECORIOLIS.ItemDeleteConfirmationTitle": "Delete Item",
   "YZECORIOLIS.ItemDeleteConfirmationContent": "Are you sure you want to delete this item?",
   "YZECORIOLIS.AutomaticFire": "Automatic fire",
-  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Number of ignored 1s (max 2)"
-}
\ No newline at end of file
+  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Number of ignored 1s (max 2)",
```

Comment:
I think we could remove that from the list no?

Reply:
true!